plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/test/index.tsx

```diff
@@ -201,12 +201,18 @@ describe( 'TotalsShipping', () => {
 				</ShippingCalculatorContext.Provider>
 			</SlotFillProvider>
 		);
+
+		const panel = screen.getByRole( 'button' );
+		const paragraph = within( panel ).getByRole( 'paragraph' );
+
 		expect(
-			screen.getByText(
-				'Delivers to W1T 4JG, London, United Kingdom (UK)'
-			)
+			within( paragraph ).getByText( ( _, element ) => {
```

Comment:
have to drill down into the button to find the text otherwise the custom matcher checking text content will match multiple times. If we don't use textContent to match then it won't match `Delivers to <strong>...</strong>`